### DISCUSSION TITLE
Pokedex -> mostrar infon en pantalla!

### DISCUSSION THREAD

**Discussion by mabvmex** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#534] [Q&A] [answered]
Hola a todos.
Necesito un poco de ayuda para poder mostrar la info de la api en la pantalla del pokedex...
Estoy intentando mostrar la información de las estadisticas y movimientos de un pokemon...

El asuto es que ya tengo información que llega a la consola del navegador pero no logro "pegarla en el HTML"
El repositorio es https://github.com/mabvmex/launch-x-S04-pokedek

Cualquier ayuda es bienvenida,
Saludos.

**Comment by Dfredude** [comment]
Me aparece que si estas accesando varias propiedades:
![image](https://user-images.githubusercontent.com/72330887/160254783-2a3d3b22-33d4-4de2-8a07-1296fdd9aa77.png)
Solo es cuestion de accesar correctamente las demas propiedades que quieras utilizar.

**Reply by mabvmex** [reply]
Hola, gracias. Sí tengo la info de la api y la puedo ver en la consola del navegador... hasta ahí va bien... la duda es que no las puedo pegar en los campos de texto, como está pegado el nombre o el tipo... Justo en eso estoy!

**Comment by carlos-gmx3** [comment]
Necesitas 3 cosas...
1 Una etiqueta, de preferencia con un ID donde vas a desplegar la información
![image](https://user-images.githubusercontent.com/99064239/160258979-f5ad845d-5d5a-413c-be61-78c6cf52e672.png)

2. Obtener los elementos del html en el, ya se q los guardes en un variable o los invoques directo
![image](https://user-images.githubusercontent.com/99064239/160259025-a811d7ae-ac97-4556-9ae9-65b4abaf1d22.png)

3. Con el uso de Inner HTML incrustar los datos
![image](https://user-images.githubusercontent.com/99064239/160259077-a7fc70b1-ad04-4287-8aec-fe4a85fa0dd8.png)
![image](https://user-images.githubusercontent.com/99064239/160259217-633e9431-932c-497c-aa4d-c1b72549e1f6.png)


Incluso puedes usar etiquetas, solo no olvides las backtits o comillas invertidas
![image](https://user-images.githubusercontent.com/99064239/160259152-3d17c271-d141-4cd7-8d19-a826835c61f0.png)

**Reply by mabvmex** [reply]
Gracias por tu ayuda... Me resultó útil.

**Comment by JosePabloGrr** [comment, marked as answer]
Tienes dos problemas en tu código en la función de "pokemonStats"
1. Código fuera del FOR (por eso no insertabas la información)
2. Tratabas de Insertar múltiples datos en un solo INPUT..

**Solución:**
1. Creas un div para encapsular el "input de TITULO para STATS"
![image](https://user-images.githubusercontent.com/16726985/160260130-8610159d-c1b1-45b1-a9e9-3cf433941f02.png)

2. Crear un input por cada STAT recibida.
![image](https://user-images.githubusercontent.com/16726985/160260149-95728a29-0c32-4675-b0af-2cfff5b6d1bc.png)

**Reply by mabvmex** [reply]
Gracias por tu ayuda.
Al final, hice algo aprecido, tuve que agregar varios inputs... mi idea inicial era poner todo en un solo input y lograr hacer scroll vertical...
Lo del for... :P Lo resolví a prueba y error hasta que lo hice funcionar...

Gracias por tu ayuda...
